src/unix/notbsd/linux/other/mod.rs

```diff
@@ -44,11 +57,24 @@ s! {
         pub ss_size: ::size_t
     }
 
+    pub struct addr_bnd_t {
```

Comment:
This type is only used in a private field, I don't think it needs to be public either.